ember-flight-icons/addon/components/flight-icon.hbs

```diff
@@ -10,5 +12,12 @@
   viewBox="0 0 {{this.size}} {{this.size}}"
   xmlns="http://www.w3.org/2000/svg"
 >
-  <use href='{{this.contextRootURL}}@hashicorp/ember-flight-icons/icons/sprite.svg#{{@name}}-{{this.size}}'></use>
+  {{#if @title}}
+    <title id={{this.titleId}}>{{@title}}</title>
+    <g role="presentation" class="flight-icon-g">
```

Comment:
thought: since this CSS selector is used only for testing (as far as I can see, I didn't see any CSS change) would it make sense to use a selector like `assert.dom('svg > g')` in the test?

Reply:
yes! I'll improve it.

Reply:
to clarify, seems like can remove this class, is unused now?

Reply:
```suggestion
    <g role="presentation">
```